src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InfoAboutTrack = (props) => {
```

Comment:
I think, there is no reason to show user info about track which he uploaded from own pc.